### PR TITLE
DEV2-3438: Add refresh capabilities on chat window show

### PR DESCRIPTION
**Pull request by ofekby**

### CODE REVIEW

File: Common/src/main/java/com/tabnineCommon/chat/ChatToolWindowListener.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    private var isLoggedIn = false
+    private var lastForceRefreshCapabilities = System.currentTimeMillis()
+
+    init {
+        ApplicationManager.getApplication().messageBus.connect()
+            .subscribe(
+                BinaryStateChangeNotifier.STATE_CHANGED_TOPIC,
+                BinaryStateChangeNotifier { isLoggedIn = it.isLoggedIn ?: false }
```
Comment (dimacodota) [inline Common/src/main/java/com/tabnineCommon/chat/ChatToolWindowListener.kt:22]: where is it used ?

Comment (ofekby) [reply Common/src/main/java/com/tabnineCommon/chat/ChatToolWindowListener.kt:22]: In the handle tabnine toolWIndow shown, the object property maintains the current logged in status by the topic, to be used when the tabnine chat toolWindow is shown.

Comment (dimacodota) [reply Common/src/main/java/com/tabnineCommon/chat/ChatToolWindowListener.kt:22]: cool

**Review by dimacodota — APPROVED** [review]
lgtm

**Pull request merged**